snappy_wrappers/wrappers/mbcs/wrapper.py

```diff
@@ -53,7 +53,7 @@ def pair_fastq_files(input_left, input_right):
 input_left = snakemake.params.args["input"]["reads_left"]
 input_right = snakemake.params.args["input"].get("reads_right", "")
 
-config = snakemake.config["step_config"]["ngs_mapping"]["somatic"]
+config = snakemake.config["step_config"]["ngs_mapping"]["mbcs"]
```

Comment:
for now, we can do it like this, and then later move those config accesses to the parameters